NOTICE.txt

```diff
@@ -0,0 +1,8 @@
+The Velox Project
```

Comment:
cc: @pedroerp

Reply:
All good, we discussed with @assignUser to create something like this. Thanks @tigrux for getting this started.

@bikramSingh91 (oncall) let's go ahead and get this one merged

Reply:
@tigrux Do you mind adding a newline at the end to this file. Internal linter is blocked on it. Thanks.

linter message:
```
Files must end in a newline.
```